### PR TITLE
COMP: MINIC: Update CMake policies to 3.10

### PR DESCRIPTION
**Pull request by blowekamp**
## PR Checklist
- [ ] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [ ] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [ ] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [ ] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [ ] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

**Comment by blowekamp** [comment]
Upstream pull request: https://github.com/BIC-MNI/libminc/pull/127

File: Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Andrew Janke - a.janke@gmail.com
 # Vladimir S. FONOV - vladimir.fonov@gmail.com
 
+CMAKE_MINIMUM_REQUIRED(VERSION 3.6...3.10)
+
```
[the remark this answers is not in the document]
Comment (N-Dekker) [reply Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt:7]: For the record: CMake version 3.10 was released already seven years ago, November 20, 2017 https://www.kitware.com/cmake-3-10-0-available-for-download

[the remark this answers is not in the document]
Comment (blowekamp) [reply Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt:7]: The change in ITK is the same as submitted upstream. The change upstream is intended to be a bug fix suitable for a patch release. Generally, dropping support for tools is not a patch release change unless necessary.

[the remark this answers is not in the document]
Comment (N-Dekker) [reply Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt:7]: I see, thanks for explaining, @blowekamp Of course, I don't know if it is really the intention of the MINC toolkit to still support those ancient CMake versions (version 3.6 to 3.9). Anyway, your PR conflicts with the one by @seanm, which does simply do `CMAKE_MINIMUM_REQUIRED(VERSION 3.10)`: https://github.com/BIC-MNI/libminc/pull/126/commits/6788cebf8f7e2c0d0229f760bb0c0df3d0bee318

Should ITK wait until either your https://github.com/BIC-MNI/libminc/pull/127 or Sean's https://github.com/BIC-MNI/libminc/pull/126 is merged to BIC-MNI/libminc?

[the remark this answers is not in the document]
Comment (blowekamp) [reply Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt:7]: yes, I have a few of these PR. Each a draft with linked changes upstream.

[the remark this answers is not in the document]
Comment (vfonov) [reply Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt:7]: > Should ITK wait until either your https://github.com/BIC-MNI/libminc/pull/127 or Sean's https://github.com/BIC-MNI/libminc/pull/126 is merged to BIC-MNI/libminc?

I already merged it

[the remark this answers is not in the document]
Comment (N-Dekker) [reply Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt:7]: > I already merged it

@vfonov Thanks , I see, on the develop branch: https://github.com/BIC-MNI/libminc/commit/6840ef7c579fa6d5f802874b7f0b2f965f09ac61

**Comment by gdevenyi** [comment]
PR title, MINIC -> MINCIO ?

**Comment by blowekamp** [comment]
@gdevenyi  This PR will be abandoned. The MINC/UpdateFromUpstream need to be modified to the correct version, and run. Last it was updated to the master branch not the develop. When do you expected these changes to make it into master?

**Pull request closed without merging**